### DISCUSSION TITLE
Importing scripts with the main code

### DISCUSSION THREAD

**Discussion by kuleshov01** [GrapesJS/grapesjs#5197] [Ideas]
Hello everyone

I want to add the page code via the import button, but when this button is clicked, my <script>code</script> disappears.
What needs to be changed so that the system allows the insertion of <script>

Working example https://github.com/GoodPHP/whitener-preset-grapesjs
![image](https://github.com/pepiuox/PHP-GrapesJS/assets/62784748/6ba73653-d6bf-456d-93da-cf938e195a1d)
![image](https://github.com/pepiuox/PHP-GrapesJS/assets/62784748/81186bd5-39b5-4e67-90c9-2b68f6152c42)
![image](https://github.com/pepiuox/PHP-GrapesJS/assets/62784748/b6191345-8175-4eb1-8d47-c23e051e12ae)

And you come out like this:
![image](https://github.com/pepiuox/PHP-GrapesJS/assets/62784748/52face1e-7bd2-4b3d-b00d-185952e12fe2)
![image](https://github.com/pepiuox/PHP-GrapesJS/assets/62784748/ca5a76f1-ed90-4a23-bf6a-9a2b3c460b63)

Disappearing!

**Comment by artf** [comment]
https://github.com/GrapesJS/grapesjs/discussions/5099
